cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/credential-step/CredentialProfile.scss

```diff
@@ -0,0 +1,14 @@
+.description {
+    margin: 25px 0;
+    line-height: 24px;
+    font-size: 14px;
+}
+
+.select-sm-width {
+    min-width: 143px;
+}
+
+.blue-text {
```

Comment:
See index.scss, there is a 'text-blue' class which already exists. You should be able to add the text-decoration: none; property to that class

Reply:
css issues will be handled in another PR.